From 6ad350b06cab8a503e7f6bcf2708959db28598c1 Mon Sep 17 00:00:00 2001
From: japm48 <japm48@users.noreply.github.com>
Date: Tue, 14 Apr 2020 15:48:06 +0200
Subject: [PATCH 11/32] grc: use correct labels in log window context menu

Previously 'gtk-clear' and 'gtk-save' were shown.

Gtk.STOCK_* constants are deprecated in favor of translated strings.
But grc is English-only, so it makes sense to stick to the simple
hardcoded solution.
---
 grc/gui/Dialogs.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/grc/gui/Dialogs.py b/grc/gui/Dialogs.py
index fd443b84da..0831ec1bd1 100644
--- a/grc/gui/Dialogs.py
+++ b/grc/gui/Dialogs.py
@@ -146,11 +146,11 @@ class TextDisplay(SimpleTextDisplay):
         lock.set_active(self.scroll_lock)
         lock.connect('activate', self.scroll_back_cb, view)
 
-        save = Gtk.ImageMenuItem(label = Gtk.STOCK_SAVE)
+        save = Gtk.ImageMenuItem(label = "Save Console")
         menu.append(save)
         save.connect('activate', self.save_cb, view)
 
-        clear = Gtk.ImageMenuItem(label = Gtk.STOCK_CLEAR)
+        clear = Gtk.ImageMenuItem(label = "Clear Console")
         menu.append(clear)
         clear.connect('activate', self.clear_cb, view)
         menu.show_all()
-- 
2.20.1

